driver/src/models/state.rs

```diff
@@ -1,26 +1,13 @@
-use byteorder::{BigEndian, LittleEndian, WriteBytesExt};
+use byteorder::{LittleEndian, WriteBytesExt};
 use serde_derive::{Deserialize, Serialize};
+use serde_json::json;
 use sha2::{Digest, Sha256};
 use web3::types::H256;
-use crate::price_finding::Solution;
-use crate::price_finding::linear_optimization_price_finder::{account_id, token_id};
 use std::collections::HashMap;
-use serde_json::json;
-
-pub const TOKENS: u8 = 30;
-pub const DB_NAME: &str = "dfusion2";
-
-pub trait RollingHashable {
-    fn rolling_hash(&self) -> H256;
-}
-
-pub trait RootHashable {
-    fn root_hash(&self, valid_items: &[bool]) -> H256;
-}
 
-pub trait Serializable {
-    fn bytes(&self) -> Vec<u8>;
-}
+use crate::models::{TOKENS, RollingHashable, Order};
+use crate::price_finding::{Solution};
+use crate::price_finding::linear_optimization_price_finder::{account_id, token_id};
```

Comment:
could we, at some point, put this account and token id into some utility file?

Reply:
In my next PR I will make those private again. The model should not know about how the linear optimization solver decides to convert IDs into strings.

Reply:
Cool beans!